### PR TITLE
Create extra type to provide way of true string quoting.

### PR DESCRIPTION
**Pull request by bacek**
Heredocs approach is fine in many cases, but it's adds newline at the
end of the original string. This causes invalid DKIM record

### CODE REVIEW

**Comment by bacek** [comment]
I'm not really happy about naming though.

**Review by paul-thompson-helix — APPROVED** [review]

File: core/core.ts [hunk 1]
```diff
@@ -128,6 +128,10 @@ export function stringValue(value: string): ResourceValue {
   return { kind: 'text', text: quotedText(value) };
 }
 
+export function quotedStringValue(value: string): ResourceValue {
```
Comment (barryskal) [inline core/core.ts:131]: Could you please add a comment to this function explaining why a user may need to call this as opposed to `stringValue` include details about the newline thing

**Review by barryskal — APPROVED** [review]
LGTM%

File: core/core.ts (hunk 1 above)
Comment (bacek) [reply core/core.ts:133]: Added

**Pull request merged**